internal/versions/versions.go

```diff
@@ -115,21 +118,34 @@ func (h *handler) GetReleaseImage(ctx context.Context, openshiftVersion, cpuArch
 	if err := h.kubeClient.List(ctx, clusterImageSets); err != nil {
 		return nil, err
 	}
+	var wg sync.WaitGroup
```

Comment:
Just for clarity, would be nice to have a comment here that the rest of the func is essentially just for caching purposes.

Reply:
Added